### ISSUE TITLE
ImportError: Could not load libtrax

### ISSUE THREAD

**Issue by alshebli9** [votchallenge/toolkit#88]
I'm trying to initialize vot and I'm getting the following error:
(vot) C:\>vot initialize
Traceback (most recent call last):
  File "D:\Anaconda\envs\vot\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "D:\Anaconda\envs\vot\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\Anaconda\envs\vot\Scripts\vot.exe\__main__.py", line 4, in <module>
  File "D:\Anaconda\envs\vot\lib\site-packages\vot\utilities\cli.py", line 9, in <module>
    from ..tracker import Registry, TrackerException
  File "D:\Anaconda\envs\vot\lib\site-packages\vot\tracker\__init__.py", line 14, in <module>
    from vot.dataset import Frame
  File "D:\Anaconda\envs\vot\lib\site-packages\vot\dataset\__init__.py", line 11, in <module>
    from trax import Region
  File "D:\Anaconda\envs\vot\lib\site-packages\trax\__init__.py", line 7, in <module>
    from ._ctypes import \
  File "D:\Anaconda\envs\vot\lib\site-packages\trax\_ctypes.py", line 503, in <module>
    _libs["libtrax"] = load_library("libtrax")
  File "D:\Anaconda\envs\vot\lib\site-packages\trax\_ctypes.py", line 191, in __call__
    raise ImportError("Could not load %s." % libname)
ImportError: Could not load libtrax.

Any idea how to fix it?

**Comment by lukacu** [comment]
See forum to continue discussion, it will be hard to follow two threads.
